bin/main.rb

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+# rubocop:disable Metrics/BlockLength
+require_relative '../lib/game'
+game = Tictactoe.new
+loop do
+  system('clear')
```

Comment:
- [ ] Your game is trying to load a file that does not exist. Please take a look at your project again and include all the files needed to run your game.

Reply:
i have added all the needed files. thank you for the feedback